Rakefile

```diff
@@ -1,2 +1,11 @@
 require "bundler/gem_tasks"
 
+# I don't know how to do this with rspec, but
+# you'll want to change this so that if someone types:
+#
+# rake
+#
+# It runs the tests by default.
+#
+# Then you can pass this off to something like TravisCI 
+# and it will work.
```

Comment:
When I pulled your gem, my first thought was to run the tests.  I knew enough to know how to see rspec and run rspec, but I'd recommend making your default rake task a full test run.

Others use this convention, including TravisCI.  By default, you push a gem up there and it will run ```rake``` to run your tests.